### ISSUE TITLE
Ensure all imported Sass partials start with an underscore.

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/brand-patterns#74]
When compiling Sass for another project, we saw an edge case where files like:

`plugins-dir.scss`
`base-dir.scss`
`modules-dir.scss`
`layouts-dir.scss`
`custom-dir.scss`

...needed to be renamed to include underscores at the beginning, like so:

`_0-plugins-dir.scss`
`_1-base-dir.scss`
`_2-modules-dir.scss`
`_3-layouts-dir.scss`
`_4-custom-dir.scss`

Not having the underscore at the beginning of the file name caused `gulp-sass` to error by not finding any mixins or variables from other directories. 

Once the underscore was added, everything worked as expected. 

tl;dr We need to rename our psuedo-manifest files to include an underscore at the beginning. OR we could move all of our imports into `styles.scss`.

**Comment by adamjohnson** [comment]
I went with moving all of our imports into `styles.scss`.
